sqlengine/mysql_engine.go

```diff
@@ -141,8 +140,52 @@ func (d *MySQLEngine) DropUser(bindingID string) error {
 }
 
 func (d *MySQLEngine) ResetState() error {
-	// TODO: Not implemented
-	return errors.New("Not implemented")
+	d.logger.Debug("reset-state.start")
```

Comment:
The Lager way of doing this is

```
logSession := d.logger.Session("reset-state")
logSession.Debug("start")
```

That produces a message/action like `things.coming.before.it.reset-state.start`

We don't usually use the Debug level either, because we want these messages to appear in our logs so we can debug it later down the life. We typically use the Info level.

Reply:
Ok I just copy-pasta'd this from the postgres implementation, like much of it TBH.

Reply:
Urgh, looking into this, making it work this way would be out of kilter with the way logging is done everywhere else in the broker, and most other methods don't log entry and exit, so I might just remove it. And someone can look at overhauling the log messages in the future if that's what they want.